src/renderer/components/cipher-inputs.vue

```diff
@@ -50,7 +50,7 @@
 
   .cipher-input {
     resize: none;
-    height: 300px;
+    height: 150px;
```

Comment:
What do you think of making this `30 vh`. Testing it out on my computer it seems to work well at multiple different view sizes. With the fixed height of 150px it doesn't look so great on big screens.

![screen shot 2018-04-12 at 6 04 59 pm](https://user-images.githubusercontent.com/17623706/38706775-7b6ba254-3e7c-11e8-90eb-5567b57c66ea.png)